.github/workflows/test_utils.yml

```diff
@@ -0,0 +1,45 @@
+name: Utils / Python - Test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macos-13]
+        python-version: [3.8, 3.9]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@main
```

Comment:
why is it needed ?

Reply:
also do you think we can reduce the model / dataset size used during testing https://github.com/huggingface/optimum/blob/main/tests/utils/test_dummpy_input_generators.py#L38 ? (using tiny random models for example)